package.json

```diff
@@ -77,15 +77,15 @@
     "github-markdown-css": "^4.0.0",
     "glob": "^7.1.3",
     "jest": "^24.5.0",
-    "jest-playwright-preset": "^1.4.2",
+    "jest-playwright-preset": "1.7.0",
     "lerna": "^3.20.2",
     "marked": "^1.2.4",
     "mkdirp": "^0.5.1",
     "mutationobserver-shim": "^0.3.3",
     "mv": "^2.1.1",
     "node-sass": "^4.13.1",
     "outdent": "^0.7.0",
-    "playwright-chromium": "^1.9.2",
+    "playwright-chromium": "1.17.1",
```

Comment:
I updated because I ran into issues that were away from the documentation. I had to adapt the footer tests to make it work, otherwise the tests I wrote would not have.